### PR TITLE
Add database table prefix

### PR DESCRIPTION
**Pull request by FJEANNOT**
**Description of the change**
Addition of a environment variable PRESTASHOP_DATABASE_PREFIX to configure the database table prefix (using the --prefix parameter of the prestashop cli installation).

**Benefits**
The use of this parameter allow users to add further configuration to their database installation, which can be benefic when migrating from a legacy system or simply to add more security against SQL injections.

**Possible drawbacks**

**Applicable issues**
Solves #103

**Additional information**

### CODE REVIEW

**Comment by carrodher** [comment]
Thanks for your contribution, I will review the changes. Apart from that, I will need to launch some tests in our internal test & release pipeline to ensure everything is working as expected in the container but also in the Helm Chart, just in case something like chart upgrades from the previous version doesn't work as expected with this change.

As the weekend is coming, I'll be back with more information at the beginning of next week

File: 1.7/debian-10/rootfs/opt/bitnami/scripts/libprestashop.sh [hunk 1]
```diff
@@ -115,6 +115,7 @@ prestashop_initialize() {
             "--db_password=${db_pass}"
             "--db_server=${db_host}:${db_port}"
             "--db_name=${db_name}"
+            "--prefix=${$PRESTASHOP_DATABASE_PREFIX}"
```
Comment (carrodher) [inline 1.7/debian-10/rootfs/opt/bitnami/scripts/libprestashop.sh:118]:
```suggestion
            "--prefix=${PRESTASHOP_DATABASE_PREFIX}"
```

**Review by carrodher — CHANGES_REQUESTED** [review]
In the `1.7/debian-10/rootfs/opt/bitnami/scripts/libprestashop.sh` file, there is a place where the prefix is hardcoded and used (around line 163), can you change it?
```diff
-mysql_remote_execute "${mysql_execute_args[@]}" <<< "UPDATE ps_configuration SET value='${setting#*=}' WHERE name='${setting%=*}';"
+mysql_remote_execute "${mysql_execute_args[@]}" <<< "UPDATE ${PRESTASHOP_DATABASE_PREFIX}configuration SET value='${setting#*=}' WHERE name='${setting%=*}';"
```

File: 1.7/debian-10/rootfs/opt/bitnami/scripts/libprestashop.sh (hunk 1 above)
Comment (FJEANNOT) [reply 1.7/debian-10/rootfs/opt/bitnami/scripts/libprestashop.sh]: Done! I think we are good to go

**Review by carrodher — APPROVED** [review]
LGTM, thanks

**Pull request merged**